app/constants/chartTypeConfigs/scatterChart.js

```diff
@@ -11,3 +17,24 @@ export const config = {
 export const defaultOpts = {
   showLegend: true,
 };
+
+/**
+ * Defines size of points for scatter chart
+ */
+const scatterChartPointRange = [60, 60];
+
+/**
+ * Similar to default opts but dependent on state
+ * called by conditionalChartOptions middleware
+ * @param {Object} state Redux application state
+ *   {Object} transformedData State deconstructed
+ * @return {Object} Object to merge into chart options, may be empty
+ */
+export const getConditionalOpts = ({ transformedData }) => {
+  if (!scatterDataHasSizes(transformedData.nvd3ScatterMultiSeries)) {
+    // Set point size for scatter charts
```

Comment:
It's kind of awkward that you have two comments referencing point **size**, when the labels are point **range**.